src/redux/reducers/books.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
```

Comment:
Please ensure that `linters` are not disabled.  A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).

Reply:
done